compio-quic/Cargo.toml

```diff
@@ -0,0 +1,81 @@
+[package]
+name = "compio-quic"
+version = "0.1.0"
+description = "QUIC for compio"
+categories = ["asynchronous", "network-programming"]
+keywords = ["async", "net", "quic"]
+edition = { workspace = true }
+authors = { workspace = true }
+readme = { workspace = true }
+license = { workspace = true }
+repository = { workspace = true }
+
+[package.metadata.docs.rs]
+all-features = true
+rustdoc-args = ["--cfg", "docsrs"]
+
+[dependencies]
+# Workspace dependencies
+compio-io = { workspace = true }
+compio-buf = { workspace = true }
+compio-log = { workspace = true }
+compio-net = { workspace = true }
+compio-runtime = { workspace = true, features = ["time"] }
+
+quinn-proto = "0.11.3"
+rustls = { workspace = true, features = ["ring"] }
```

Comment:
Move `ring` feature into dev-dependencies.

Reply:
`ring` is implicitly enabled by `quinn-proto` (default features) and a [PR](quinn-rs/quinn#1962) for supporting `aws-lc-rs` is pending.

Reply:
That's the problem of quinn-proto, and we shouldn't produce such problems again.

Reply:
Address this one, please. Either move the feature into dev-dependencies, or just remove the feature.